frontend/packages/vsphere-plugin/src/components/ClusterOverview/VSphereStatus.css

```diff
@@ -0,0 +1,4 @@
+.plugin-vsphere-status-popup {
+    max-width: 30rem !important;
+    width: 30rem;
```

Comment:
I think it does not make sense to define both ? `width` should be enough, right ?

Reply:
This line is overriding it: https://github.com/openshift/console/blob/master/frontend/packages/console-shared/src/components/dashboard/status-card/HealthItem.tsx#L58

Even with `!important`, the result is not as expected when just a single attribute is set here.

Reply:
We can make it more generic there but this is fast workaround. I can add a comment to fix later.